build-gluster-org/jobs/user-update.yml

```diff
@@ -0,0 +1,36 @@
+- job:
+    name: update-admin-list
+    node: 
+    description: This job updates the jenkins job's admin-list with the members of gluster-maintainers 
+    project-type: freestyle
+    concurrent: true
+
+    scm:
+    - git:
+        url: git@github.com:gluster/build-jobs.git
+        credentials-id: github-credentials
+        branches:
+        - remotes/origin/master
+        basedir: "build-jobs"
+
+    builders:
+    - shell: |
+        cat << EOF > requirements.txt
+        gitdb==4.0.5
+        GitPython==3.1.9
+        EOF
+        pip install -r requirements.txt
+    
+    - python: !include-raw: ../scripts/user-update.py  
+
+    parameters:
+    - string:
+        description: 'Enter the job names (space seperated)' 
+        name: JOB_DETAILS 
+
+    wrappers:
+    - credentials-binding:
+         - username-password-separated:
+            credential-id: github-credentials 
+            username: GITHUB_USERNAME
+            password: GITHUB_PASSWORD
```

Comment:
That's interesting, because it seems that the other jenkins jobs do not use that. Likely a bug somewhere, or we are using some trick outside jenkins. I will dig a bit more.

And we also need a ssh key if not already done, but our secret management on jenkins is a bit messy, we have 5 differents passwords for older builders, etc, etc.

Reply:
This is the credentials of Gluster-ant which I received from Deepshikha.
https://build.gluster.org/credentials/store/system/domain/bugs_handling_systems/credential/github-credentials/ 
Basically I don't have manage jenkins permission for upstream but as per the URL the credential id is - github-credentials. (which I need to confirm as I am doubtful at the moment.) 
The same credentials will be used for pulling repo, pushing changes, fetching user-list etc.